pallets/phala-world/src/incubation.rs

```diff
@@ -32,42 +36,82 @@ pub mod pallet {
 		/// Amount of food per Era
 		#[pallet::constant]
 		type FoodPerEra: Get<u8>;
-		/// Max food an Origin of Shell can be fed per day
-		#[pallet::constant]
-		type MaxFoodFedPerEra: Get<u16>;
 		/// Max food to feed your own Origin of Shell
 		#[pallet::constant]
 		type MaxFoodFeedSelf: Get<u8>;
+		/// Duration of incubation process
+		#[pallet::constant]
+		type IncubationDuration: Get<u64>;
 	}
 
 	#[pallet::pallet]
+	#[pallet::without_storage_info]
 	#[pallet::generate_store(pub(super) trait Store)]
 	pub struct Pallet<T>(_);
 
-	/// Food per Owner/Spirit where an owner gets 5 food per era
+	/// Info on Origin of Shells that the Owner has fed
 	#[pallet::storage]
 	#[pallet::getter(fn get_food_by_owner)]
-	pub type FoodByOwner<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, FoodInfo>;
+	pub type FoodByOwner<T: Config> =
```

Comment:
```suggestion
	pub type FoodByOwners<T: Config> =
```

Reply:
The naming convention of maps suggest to make it in plural form (adopted by Parity / Polkadot)